gst/nnstreamer/tensor_repo/tensor_reposink.c

```diff
@@ -129,22 +121,13 @@ gst_tensor_reposink_class_init (GstTensorRepoSinkClass * klass)
       "Set element to handle tensor repository",
       "Samsung Electronics Co., Ltd.");
 
-#if (GST_VERSION_MAJOR == 1) && (GST_VERSION_MINOR >= 8)        /* >= 1.8 */
-  gst_element_class_add_static_pad_template (element_class, &sink_template);
-#elif (GST_VERSION_MAJOR == 1)  /* 1.0 ~ 1.8 */
-  {
-    GstPadTemplate *pad_template;
-    GstCaps *pad_caps;
-    pad_caps = gst_caps_from_string (GST_TENSOR_CAP_DEFAULT "; "
-        GST_TENSORS_CAP_DEFAULT);
-    pad_template = gst_pad_template_new ("sink", GST_PAD_SINK, GST_PAD_ALWAYS,
-        pad_caps);
-    gst_element_class_add_pad_template (element_class, pad_template);
-    gst_caps_unref (pad_caps);
-  }
-#else
-#error We support GStreamer 1.x only.
-#endif
+  /* pad template */
```

Comment:
It seems to be used repeatedly. How about making it as util function?

Reply:
is it better to keep static GstStaticPadTemplate in each element and set from this?

Reply:
Um, It seems better to define pads in each element. Thank you for your opinion.